src/docs/csharp/best-practices/code-style.md

```diff
@@ -0,0 +1,82 @@
+---
+tags:
+  - melhores práticas
+  - testes
+--- 
+# Testes
+<authors>
+    <author user="andresantarosa"/> 
+</authors>  
+
+---
+Seguir um padrão de código é essencial para facilitar a legibilidade do código e a manutentabilidade da aplicação como um todo. Em geral linguagens possuem alguns padrões definidos de como codificar certos comandos afim de padronizar a escrita entre os desenvolvedores.
+
+## var
+O `var` é uma keywork utilizada na tipagem implícita, utilizando o `var` torna-se desncessário a explitação do tipo da variável e é uma grande aliada para deixar o código mais limpo e fluído. Entretanto é necessário ter cuidado na sua utilização, visto que se utilizado fora de lugar pode piorar consideravelmente a inteligibilidade.
+
+### Não utilizar `var` quando o tipo de retorno não é explícito.
```

Comment:
I disagree with that, it's possible use `var` in all scenarios.

Using `var` in this cases should the code keeps a pattern, but it's my opinion.